qiskit_experiments/library/characterization/t1.py

```diff
@@ -106,6 +108,19 @@ def _set_backend(self, backend: Backend):
                 timing_constraints=timing_constraints, scheduling_method=scheduling_method
             )
 
+        # Set conversion factor
+        if self.experiment_options.unit == "dt":
+            try:
+                dt_factor = getattr(self.backend.configuration(), "dt")
+                conversion_factor = dt_factor
+            except AttributeError as no_dt:
+                raise AttributeError("Dt parameter is missing in backend configuration") from no_dt
+        elif self.experiment_options.unit != "s":
+            conversion_factor = apply_prefix(1, self.experiment_options.unit)
+        else:
+            conversion_factor = 1
+        self.set_experiment_options(conversion_factor=conversion_factor)
```

Comment:
I don't think it's correct that the conversion factor is now an experiment option. Experiment options are something that the user can control. Do we want to allow the user to set it, or do we determine it from the backend? Here (in this line) for example it's a fragile situation, where we override the option that the user might have explicitly set to another value.

Reply:
I think this is really temporary solution and don't want to put more effort for this. Since we decided to remove `unit` as per #456, these code will be immediately removed.